doc/source/changes.rst

```diff
@@ -6,6 +6,13 @@ Changes/Hints for migration
 This chapter will give you hints on how to handle version migration, in particular regarding what you may need
 to adapt in your project configuration, database etc. when upgrading to a new version.
 
+Version 2.6.0
+-------------
+* New parameter 'default_toc_length' allows to define a default table of content pages number avoiding a second
```

Comment:
I would omit the statemtents about the 95% and the default setting. I guess default is to not set this parameter.